src/Ushahidi/Modules/V5/Actions/Media/Handlers/CreateMediaCommandHandler.php

```diff
@@ -35,7 +35,7 @@ protected function isSupported(Command $command)
     public function __invoke(Action $action)
     {
         $this->isSupported($action);
-        $this->validateFileData($action->getMediaEntity());
+        // $this->validateFileData($action->getMediaEntity());
```

Comment:
I am disabling this because it presents a hard upload limit which is in opposition to the frontend functionality. I will be creating a new ticket for changes to the backend to handle the frontend functionality, and in the interim have implemented validation on the frontend.